### PR TITLE
Add data.tables documentation

### PR DESCRIPTION
**Pull request by Ken-Sumi1019**
Because tabels is implemented but not documented.

### CODE REVIEW

File: website/docs/d/tables.html.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: "mysql"
+page_title: "MySQL: mysql_tables"
+sidebar_current: "docs-mysql-datasource-tables"
+description: |-
+  Get tables on a MySQL server.
```
Comment (petoju) [inline website/docs/d/tables.html.markdown:6]: Just nit-picking: please add "s" as it's 3rd person singular. So it `gets` tables from a MySQL server.

The same issue is with description a bit lower

File: website/docs/d/tables.html.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  database = "my_awesome_app"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `database` - (Required) The name of the database.
+
```
Comment (petoju) [inline website/docs/d/tables.html.markdown:27]: Maybe the newline here is not necessary.

File: website/docs/d/tables.html.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* `pattern` - (Optional) Patterns for searching tables.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `tabels` - The list of the table names.
```
Comment (petoju) [inline website/docs/d/tables.html.markdown:34]:
```suggestion
* `tables` - The list of the table names.
```

**Review by petoju — CHANGES_REQUESTED** [review]
Thanks, it mostly looks good - I have only some grammar issues / extra newline / type.

**Comment by Ken-Sumi1019** [comment]
Thanks for your review.
I have corrected everything.

**Comment by petoju** [comment]
Thanks for the fixes.

**Pull request merged**